app/router.js

```diff
@@ -34,9 +34,8 @@ Router.map(function() {
   this.route('blog', {path: 'blogs/:slug'}, subpageRoutes);
   this.route('stream', {path: 'streams'});
   this.route('playlist', {path: 'streams/:slug'});
-  this.route('schedule', function() {
-    this.route('date', {path: ':year/:month/:day'});
-  });
+  this.route('schedule-redirect', { path: 'schedule' });
+  this.route('schedule', { path: 'schedule/:year/:month/:day' });
```

Comment:
I should also add that this architecture is me preparing for the future where the "date" is a query string instead of a series of year/month/day subdirs, which never made total sense to me.